lib/coherence/schema.ex

```diff
@@ -185,7 +185,7 @@ defmodule Coherence.Schema do
         Returns a changeset ready for Repo.update
         """
         def lock(user, locked_at \\ Ecto.DateTime.utc) do
-          Config.user_schema.changeset(user, %{locked_at: locked_at})
+          Config.user_schema.changeset(user, %{locked_at: locked_at, current_password: user.password})
```

Comment:
The current_password should not be required here. Please update the changeset logic so it only requires current_password validation if the password is being changed.